src/Panel/Search.php

```diff
@@ -19,15 +19,4 @@
 class Search extends Json
 {
 	protected static string $key = '$search';
-
-	public static function response($data, array $options = []): Response
-	{
-		if (is_array($data) === true) {
-			$data = [
-				'results' => $data
```

Comment:
Oh, couldn't we just use this place to polyfill old search plugins that don't define their own results and pagination keys?

Reply:
As in `if is array but not has array key "results"`?

Reply:
But how would we get the pagination for them?

Reply:
I think we need to switch off pagination for those custom searches that don't define it.

Reply:
Or we just treat it as `page: 1, limit: array.length, total: array.length`

Reply:
Great idea